test/08_string.jl

```diff
@@ -138,10 +145,10 @@ using CategoricalArrays
     @test match(r"af", v2, 2).offset === 2
     @test match(r"af", v2, 2, UInt32(0)).offset === 2
 
-    @test matchall(r"af", v1) == []
-    @test matchall(r"af", v2) == ["af"]
-    if VERSION > v"0.7.0-DEV.3526"
-        @test matchall(r"af", v2, overlap=true) == ["af"]
+    @test collect((m.match for m ∈ eachmatch(r"af", v1))) == []
+    @test collect((m.match for m ∈ eachmatch(r"af", v2))) == ["af"]
+    if VERSION > v"0.7.0-"
```

Comment:
Why remove the exact version?

Reply:
Again, now I forget what happened.  I'm pretty sure one of these caused a problem and then I got overzealous and removed all of them.